nsv13/code/modules/overmap/overmap_ghosts.dm

```diff
@@ -98,3 +101,6 @@
 
 	else //Try again later
 		addtimer(CALLBACK(src, PROC_REF(ghost_key_check), ghost), 1 SECONDS)
+
+/obj/structure/overmap/proc/spec_ghostship_changes() //Proc to buff ghost ships. Must be overridden to apply buffs
+	return
```

Comment:
But why call an empty proc?

Reply:
Standard practice for OOP (basetype proc interface that just returns without doing anything)
Proccall overhead for calling empty proc should be comparable to running an if to check for said empty proc and this solution is the cleaner one inheritance-wise.